### ISSUE TITLE
Soft incompatibility with Dye Depot: generates blocks, but no models available

### ISSUE THREAD

**Issue by obsidian29** [MehVahdJukaar/sleep_tight#83]
Hello,

I have long noticed a small issue with this mod and Dye Depot - it does successfully generate block and item pairs for the hammocks of the various Dye Depot colors, but they have no models or textures. I was going to fix it myself with a data pack, but when I checked the license and found that your mod is ARR, I realized I would not be able to use Sleep Tight's models and textures and recolor them to the Dye Depot colors, and it would look stylistically mismatched.

That being the case, I figured, maybe better make sure the dev actually knows this is happening. XD If you don't want to do it yourself and you're okay with me doing it, if you give me explicit permission to just create recolors/models for the Dye Depot colors, I would happily make a PR for you to add it to your mod. It's up to you, I won't try without permission.

Thank you for the mod, it makes sleep feel much more weighty in the game and night bags are massively useful for creative! Have a nice day.

**Comment by MehVahdJukaar** [comment]
If you want to publish a compat texture pack you are free to do som that being said this is not a compatibility issue, actually quite the opposite as the mod does add blocks for them. None of those textures are generated tho. What you are asking is dye depot textures compat which actually im pretty sure some mod does already. Adding colored textures and mod compact assets is basically a dye mod MAIN job (anybody can add new dyes to the enum) since vanilla assets are few but a good dye mod shows it's worth when it comes to mod support as adding a dye is an operation which is always somewhat questionable and could even break mods. So if you want you should ask for support there

**Comment by obsidian29** [comment]
Creating a compat texture pack with models and textures was my first idea, yes. The reason I came to you and not to Dye Depot is because your mod is ARR, which I completely agree with - all my mods are ARR too - but it meant that I could not go ahead with creating that resource pack without your explicit permission.

If I do in fact have that permission now to create recolored textures and models in a resource pack for the generated blocks, I would happily do so and will upload it to Modrinth for others to use, with your permission. I do want explicit written permission though, or I won't go ahead. I know I would not want someone to try to use or recolor any of my work without that.

I'll wait to hear from you before moving forward, and thanks for replying!

**Comment by MehVahdJukaar** [comment]
yeah as ive said you have permissions to make such a pack. I should probably update my custom license toexplicitly allow such cases. Obviously redistributing mod assets would be against but not when used for a thing made *for* the mod itself
